.gitignore

```diff
@@ -2,9 +2,10 @@
 *.idea
 *.DS_Store
 *.vscode
-*.csv
 *.pt
 *.json
 *.sqlite3
 chemprop.egg-info
 docs/build
+conformal/
+test_plan/
```

Comment:
These are directories on your local setup right? We don't need to add them to the gitignore in this PR. Would also keep the csv exclusion.